pkg/apis/core/validation/controllerregistration_test.go

```diff
@@ -100,7 +100,7 @@ var _ = Describe("validation", func() {
 			resource := core.ControllerResource{
 				Kind:            v1alpha1.ExtensionResource,
 				Type:            "arbitrary",
-				GloballyEnabled: test.MakeBoolPointer(true),
+				GloballyEnabled: framework.MakeBoolPointer(true),
```

Comment:
Why not use https://github.com/gardener/gardener/tree/master/vendor/k8s.io/utils/pointer ?